test/safe.ts

```diff
@@ -0,0 +1,56 @@
+import GnosisSafe from "@gnosis.pm/safe-contracts/build/artifacts/contracts/GnosisSafe.sol/GnosisSafe.json";
```

Comment:
These seem to be the 1.2.0 contract. With 1.3.0 there will also be an event that is emitted in the `receive` case. Is this taken into consideration?

Reply:
Note: you could pull the ABI from https://github.com/gnosis/safe-deployments but we are currently missing the deployment code in the deployments which might be interesting for tests ... I opened an issue for this: https://github.com/gnosis/safe-deployments/issues/8

Reply:
Edit: was not even aware that we include the hardhat build artifacts in the npm release :grimacing: